Ladybird/ConsoleWidget.cpp

```diff
@@ -159,9 +159,13 @@ void ConsoleWidget::print_html(StringView line)
         document.body.appendChild(p);
 )~~~"sv);
 
-    // FIXME: Make it scroll to the bottom, using `window.scrollTo()` in the JS above.
-    //        We used to call `m_output_view->scroll_to_bottom();` here, but that does not work because
-    //        it runs synchronously, meaning it happens before the HTML is output via IPC above.
+    // It should be sufficient to scrollTo a y value of document.documentElement.offsetHeight,
+    // but due to an unknown bug offsetHeight seems to not be properly updated after spamming
+    // a lot of document changes.
+    //
+    // The 1ms setTimeout makes the scrollTo async and allows the DOM to be updated.
```

Comment:
That seems unnecessary, `setTimeout` with a 0 delay is still queued and runs asynchronously